src/widgetastic/widget.py

```diff
@@ -374,7 +374,11 @@ def root_browser(self):
 
     @property
     def parent_browser(self):
-        return self.parent.browser
+        try:
+            return self.locatable_parent.browser
```

Comment:
Using `hasattr()` pattern elsewhere, why not here? or are you catching AttributeError on `locatable_parent` and on `locatable_parent.browser` both?